rs/config/src/embedders.rs

```diff
@@ -39,6 +39,9 @@ pub(crate) const DEFAULT_COST_TO_COMPILE_WASM_INSTRUCTION: NumInstructions =
 /// The number of rayon threads used by wasmtime to compile wasm binaries
 const DEFAULT_WASMTIME_RAYON_COMPILATION_THREADS: usize = 10;
 
+/// The number of rayon threads use for the parallel page copying optimization.
+const DEFAULT_PAGE_ALLOCATOR_THREADS: usize = 8;
```

Comment:
Out of curiosity, how are these numbers of 8 and 10 chosen?

Reply:
They're both chosen from benchmarks. Both are "sweetspots" for the two operations. If we increase the numbers some more we see some diminishing returns.